src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -54,10 +54,10 @@ addLine(Json::Value& jsonLines, RPCTrustLine const& line)
         jPeer[jss::authorized] = true;
     if (line.getAuthPeer())
         jPeer[jss::peer_authorized] = true;
-    if (line.getNoRipple() || !line.getDefaultRipple())
-        jPeer[jss::no_ripple] = line.getNoRipple();
-    if (line.getNoRipplePeer() || !line.getDefaultRipple())
-        jPeer[jss::no_ripple_peer] = line.getNoRipplePeer();
+    if (line.getNoRipple())
+        jPeer[jss::no_ripple] = true;
+    if (line.getNoRipplePeer())
+        jPeer[jss::no_ripple_peer] = true;
```

Comment:
This should probably delete the `TrustLineBase::getDefaultRipple()` function, too.